test/helpers/deploy/deploy-zns.ts

```diff
@@ -164,25 +168,27 @@ export const deployDomainToken = async (
   return domainToken as unknown as ZNSDomainToken;
 };
 
-export const deployMeowToken = async (
+export const deployZToken = async (
   deployer : SignerWithAddress,
+  governorAddresses : Array<string>,
+  adminAddresses : Array<string>,
   isTenderlyRun : boolean
-) : Promise<MeowTokenMock> => {
-  const factory = new MeowTokenMock__factory(deployer);
-
-  const meowToken = await hre.upgrades.deployProxy(
-    factory,
-    [
-      meowTokenName,
-      meowTokenSymbol,
-    ],
-    {
-      kind: "transparent",
-    }
-  ) as unknown as MeowTokenMock;
-
-  await meowToken.waitForDeployment();
-  const proxyAddress = await meowToken.getAddress();
+) : Promise<ZTokenMock> => {
+  const Factory = new ZTokenMock__factory(deployer);
+  const zToken = await Factory.deploy(
+    zTokenName,
+    zTokenSymbol,
+    governorAddresses[0],
+    INITIAL_ADMIN_DELAY_DEFAULT,
+    deployer.address,
+    adminAddresses[0],
+    INITIAL_SUPPLY_DEFAULT,
+    INFLATION_RATES_DEFAULT,
+    FINAL_INFLATION_RATE_DEFAULT
+  ) as unknown as ZTokenMock;
```

Comment:
We don't need `unknown` here. Look at the output from the above `factory,deploy()` and you'll see that it returns an object of type `ZTokenMock` already.

The `as unkown as MyType` syntax is basically a way to tell the compiler to not complain when you're intentionally breaking type safety. If you aren't doing that, you don't need it.